plugin/docker/containersDir.go

```diff
@@ -0,0 +1,48 @@
+package docker
```

Comment:
This kind of rename I like to break out into a separate commit. But not worth redoing now.